app/liquid_tags/link_tag.rb

```diff
@@ -33,7 +34,7 @@ def article_hash(slug)
     path.slice!(0) if path.starts_with?("/") # remove leading slash if present
     path.slice!(-1) if path.ends_with?("/") # remove trailing slash if present
     extracted_hash = Addressable::Template.new("{username}/{slug}").extract(path)&.symbolize_keys
-    raise StandardError, "The article you're looking for does not exist: {% link #{slug} %}" unless extracted_hash
+    raise StandardError, "The article you're looking for does not exist: #{slug}" unless extracted_hash
```

Comment:
I changed this error message because I feel the main piece of information is the link that was passed in, not the syntax for the liquid tag.